externs/shaka/net.js

```diff
@@ -118,6 +118,8 @@ shaka.extern.Request;
  *   The original URI passed to the browser for networking. This is before any
  *   redirects, but after request filters are executed.
  * @property {BufferSource} data
+ *   The response HTTP status code.
```

Comment:
I think the comment is inverted, this should go in status and the status in data.

Reply:
Omg! I am so sorry for this, fixed!